packages/react-devtools-shared/src/backend/renderer.js

```diff
@@ -111,6 +111,11 @@ type ReactTypeOfSideEffectType = {|
   Placement: number,
 |};
 
+function getFiberFlags(fiber: Fiber) {
+  // The name of this field changed from "flags" to "flags"
```

Comment:
From "flags" to "flags" 😄

Reply:
Super tiny nit, mostly just thinking out loud: Since this function would be called in a relative hot path, I wonder if we could reduce the overhead of an added `typeof` check by leveraging a technique like `getInternalReactConstants` that caches the lookup result, e.g.

```js
const getFiberFlags = SOME_HEURISTIC_OTHER_THAN_VERSION_NUMBER
  ? fiber => fiber.flags
  : fiber => fiber.effectTag;
```

Reply:
Yeah I'm cool with that. I wasn't sure what the implications might be for multiple copies of React. How will that work?

Reply:
Each copy of React (each renderer really) will injected itself separately. Conditional methods like this just need to be declared inside of the `attach` function, so they're scoped to a specific renderer instance/version.

Reply:
I'm not actually sure if the VM will be able to inline a dynamic function, so there's a chance it might actually make it slower: function call in hot path instead of type check.

Reply:
Interesting.

This was a nit anyway so feel free to ignore it :)